details.html

```diff
@@ -0,0 +1,141 @@
+              <td class="ingredientAmount">1 spoon</td>
+              <td class="ingredientText">fish or soya sauce</td>
+            </tr>
+          </table>
+        </div>
+        <div class="splitview-second">
+          <h2 class="detail-header">Preparation</h2>
+          <ol class="preparationList">
+            <li itemprop="recipeIntructions" class="preparationStep">
+              <span
+                >Have a look on your fridge, choose all ingredients you want and
+                get it ready for cooking.
+              </span>
+            </li>
+            <li itemprop="recipeIntructions" class="preparationStep">
+              <span
+                >Prepare noodles based on instructions and cut all vegatable as
+                you want.
+              </span>
+            </li>
+            <li itemprop="recipeIntructions" class="preparationStep">
+              <span
+                >Boiled water and add spring onion, galic, vegetables and spices
+                like white pepper, dashi powder, soy sauce, ect. Cook until
+                vegetables are well cooked.</span
+              >
+            </li>
+            <li itemprop="recipeIntructions" class="preparationStep">
+              <span
+                >Serve noodle in a bowl with herbs and add hot soup before
+                serving.</span
+              >
+            </li>
+          </ol>
+        </div>
+      </div>
+      <div class="commentcontainer">
+        <div class="comment1">
+          <h2 class="commenttitle">What people say</h2>
+        </div>
+        <div class="comment2">
+          <div class="commentblock">
+            <div class="commentname">Hans Muster</div>
+            <div class="commentparagraph">
+              <p>
+                "Love this recipes, very simple and quickly"
+              </p>
+            </div>
+          </div>
+          <div class="commentblock">
+            <div class="commentname">Peter Muster</div>
+            <div class="commentparagraph">
+              <p>
+                "I try this recipe and it is very simple so that I could do it when my wife is not at home"
+              </p>
+            </div>
+          </div>
+          <div class="commentblock">
+            <div class="commentname">Franz Muster</div>
+            <div class="commentparagraph">
+              <p>
+                "This recipe helps you to avoid food waste"
+              </p>
+            </div>
+          </div>
+        </div>
+      <form name="comment" method="post" netlify>
+        <h2 class="detail-header">Add comments</h2>
+        <label>
+          <input
+            type="text"
+            name="name"
+            placeholder="Name"
+            required
+            maxlength="8"
+          />
+        </label>
+        <label>
+          <input
+            type="text"
+            name="comment"
+            placeholder="Comment"
+            required
+            maxlength="500"
+          />
```

Comment:
Maybe if you want the comments to be longer, this could be a textarea instead of input with type text? 😇